stripe/src/main/java/com/stripe/android/IssuingCardEphemeralKey.java

```diff
@@ -0,0 +1,76 @@
+package com.stripe.android;
+
+import android.os.Parcel;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+class IssuingCardEphemeralKey extends AbstractEphemeralKey {
+
+    protected IssuingCardEphemeralKey(Parcel in) {
+        super(in);
+    }
+
+    protected IssuingCardEphemeralKey(
+            long created,
+            @NonNull String issuingCardId,
+            long expires,
+            @NonNull String id,
+            boolean liveMode,
+            @NonNull String object,
+            @NonNull String secret,
+            @NonNull String type
+    ) {
+        super(created,
+                issuingCardId,
+                expires,
+                id,
+                liveMode,
+                object,
+                secret,
+                type);
+
+    }
+
+    @SuppressWarnings("checkstyle:RedundantModifier") // Not actually redundant :|
```

Comment:
I think you can remove this annotation, I'm not seeing any issues when it's removed

Reply:
Ah let me try that

Reply:
ah so `checkstyle lint` raise an ERROR if we don't add it :|

Reply:
This is redundant because the class isn't `public`, so nothing in the public scope would be able to instantiate this class. You can change the modifier to `protected` and remove the annotation.

Reply:
thanks for the clarification